<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stock</class>
 <widget class="QWidget" name="Stock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QPushButton" name="stock_recorrect">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stock_watch">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stock_insert">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>录入</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stock_back">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入厂商名称</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
